ci/kokoro/windows/build-dependencies.ps1

```diff
@@ -110,6 +110,10 @@ foreach ($pkg in $packages) {
     if ($LastExitCode) {
         throw "vcpkg install $pkg failed with exit code $LastExitCode"
     }
+    # Remove the buildtree directory
+    if (Test-Path t:\src\github\vcpkg\buildtrees) {
+       Remove-Item -path t:\src\github\vcpkg\buildtrees -recurse
+    }
```

Comment:
Opt: It looks like
```
Remove-Item -Path t:\src\github\vcpkg\buildtrees -Recurse -ErrorAction Ignore
```
might be the one-shot way to do that.

Reply:
Yes, that would work, but `-ErrorAction Ignore` will also ignore all the other errors, so I'd use `if` for now.

Reply:
Okie dokie.